src/components/TimeseriesChart.js

```diff
@@ -889,6 +875,52 @@ function backgroundColorBetweenTwoX(
   };
 }
 
+function mapStateToProps(state) {
+  return {
+    measuringstations: state.assets.measuringstation || {},
+    getRaster: makeGetter(state.rasters),
+    timeseries: state.timeseries,
+    rasterEvents: state.rasterEvents,
+    areRasterEventsLoaded: intersectionUuid => {
+      let shortIntersectionUuid, theRasterEventsObject;
+      if (!state.rasterEvents) {
+        return null;
+      } else {
+        for (let longUuid in state.rasterEvents) {
+          shortIntersectionUuid = longUuid.slice(0, 7);
```

Comment:
Ik vind het zo erg dat deze kennis overal door onze codebase heen zit.

Reply:
Als ik je goed begrijp dan bedoel je dat de check of bepaalde api objecten gereed zijn in meerdere componenten in de mapstatetoprops functie (of andere plekken) staat gehardcode.
Ik denk dat dit wel naar de reduxstore zou kunnen?